tvtk/pyface/tvtk_scene.py

```diff
@@ -808,7 +816,7 @@ def _create_control(self, parent):
 
         self._interactor.initialize()
         self._interactor.render()
-        self.light_manager = light_manager.LightManager(self)
+        self.light_manager = light_manager.LightManager(weakref.proxy(self))
```

Comment:
Ok. I guess I must have used `pympler console browser`  to report back references which showed LightManager, before doing a `gc.collect`. If `_closed_fired` breaks the cycle, then not keeping it a weakref makes sense.